### DISCUSSION TITLE
Did anyone try the new "Masking" ability ?

### DISCUSSION THREAD

**Discussion by uberneko** [comfyanonymous/ComfyUI#5992] [General]
That was a good news, https://blog.comfy.org/masking-and-scheduling-lora-and-model-weights/, and decided to give it a try -

With some adjustments... no checkpoint but changing with Model (Flux1.dev), CLIP and VAE loaders - also with two "Create Hook LoRA" - as I wanted to make one specific person on the right and one specific on the left with the basic black/white mask.

Although usually my 4070Ti Super takes about 30s to create a picture, it took more than 14mins with a poor result (it seems to work since I can see the two people I needed) but it is quite blurred. Reducing to less steps would be far worse.
Did you experience the same slow-down ?

**Comment by camoody1** [comment]
I've just tried the same thing... two different characters loras in Flux. In addition to taking over 10-minutes on my RTX 4070Ti, it also does not actually work. Both characters (one on each side of the screen) used the second hooked lora. I could not get the first lora to be recognized at all.

**Comment by uberneko** [comment]
Hello !
Thank you for your answer -
I actually had time to work on it a bit more.

What I understand so far is that the two prompts have to be the same and describe the whole scene, with trigger words.
I decreased the cfg to 3.5, denoise to 0.85 and I have something good - just a bit odd (for instance when I ask for a realistic picture it keeps some cartoon aspects, don't know why) - but it is much better. And a 1344 x 768 pictures takes now 7mins (using flux dev fp8).

**Comment by ColonelJay** [comment]
Hi @uberneko,
I guess I have a similar issue but may be a bit more complex. I opened a dedicated post about it here : [ComfyUI discussion](https://github.com/comfyanonymous/ComfyUI/discussions/6443)

Thank in advance for any hints about it !

**Comment by uberneko** [comment]
Hello !
Some news -
During my last message, in January, it took me usually about 8:30 mins to have an image with two LoRAs.

In March, I realized that it has been divided by almost 2, like 4:30 mins...

And I realize now, with ComfyUI 3.29, it went down again to 2:30 mins ! So again almost divided by two.

I don't know what you did guys, within the code, for Masking, but that was really great !! Thank you !
